.claude/commands/pr-review.md

```
@@ -1,22 +1,17 @@
 ---
 name: pr-review
 description:
-  Fetch CodeRabbit comments and CI check results for the current PR and write
-  them to .claude/pr-review.md
+  Fetch CodeRabbit comments and CI check results for the current PR.
 ---
 
-Run `bash scripts/pr-review-summary.sh` to collect all CodeRabbit inline
-comments, the walkthrough summary, and CI check statuses for the current
-branch's open PR.
+Run `bash scripts/pr-status-check.sh` to fetch all CodeRabbit review comments
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)